### ISSUE TITLE
Store data on only 3 nodes

### ISSUE THREAD

**Issue by WhiteTrashLord** [RedisLabs/redisraft#63]
Let's assume we have a RedisRaft cluster with 9 nodes. Currently the same data will be saved on all 9 nodes or am I wrong?
Do you plan that users can change the replication factor?
I think for most users it would be enough that the data is stored on 3 nodes. This way a cluster could save much more data.

**Comment by yossigo** [comment]
You are not wrong, this is how RedisRaft (and Raft itself) work. The goal is strong consistency and reliability that can be guaranteed as long as at least (N/2)+1 of the nodes are available.

If you're OK with N=3, you simply need to create a RedisRaft cluster of 3 nodes. If you need more compute or RAM resources for your dataset, the best you can do now is create 3 RedisRaft clusters (3 nodes each) and  partition/shard your data at the client side. In the future we may also have additional clustering options.

If you're looking for more flexible configuration and more relaxed consistency/durability guarantees then you may consider using [Redis Cluster](https://redis.io/topics/cluster-tutorial) instead.

Oh and just to be sure you're aware, RedisRaft is still under development and we don't consider it ready for production!
